src/ApiServer/ApiServer/Service/GenerationService.cs

```diff
@@ -158,14 +173,17 @@ public async Task VerifyGenerationRequest(Guid otcGen) {
                     type: "https://wom.social/api/problems/otc-not-valid"
                 );
             }
+
             if(!request.Verified) {
-                Logger.LogInformation(LoggingEvents.Operations, "Voucher generation {0} not verified, cannot be performed", request.Otc);
+                Logger.LogInformation(LoggingEvents.Operations,
+                    "Voucher generation {0} not verified, cannot be performed", request.Otc);
```

Comment:
Queste modifiche qua (ce ne sono diverse) sono generate da qualche impostazione/plug-in che forza le righe troppo lunghe ad andare a capo (immagino). Disabiliterei questa opzione e rimuoverei le modifiche in modo da ridurre le modifiche "non sostanziali" della PR.